src/helpers/credential.rs

```diff
@@ -731,4 +881,38 @@ mod tests {
 
         Ok(())
     }
+
+    #[tokio::test]
+    #[cfg(feature = "plugin-did-sidetree")]
+    async fn helper_can_create_self_issued_credential() -> Result<()> {
+        let mut vade_evan = VadeEvan::new(crate::VadeEvanConfig {
+            target: "test",
+            signer: "remote|http://127.0.0.1:7070/key/sign",
+        })?;
+        let credential_subject_str = r#"{
+            "id": "did:evan:EiAOD3RUcQrRXNZIR8BIEXuGvixcUj667_5fdeX-Sp3PpA",
+            "data": {
+                "email": "value@x.com"
+            }
+        }"#;
+        let bbs_secret = r#"GRsdzRB0pf/8MKP/ZBOM2BEV1A8DIDfmLh8T3b1hPKc="#;
+        let bbs_private_key = r#"WWTZW8pkz35UnvsUCEsof2CJmNHaJQ/X+B5xjWcHr/I="#;
+        let schema_did = r#"did:evan:EiACv4q04NPkNRXQzQHOEMa3r1p_uINgX75VYP2gaK5ADw"#;
```

Comment:
Do we need the raw string literals here? ^^

```suggestion
        let bbs_secret = "GRsdzRB0pf/8MKP/ZBOM2BEV1A8DIDfmLh8T3b1hPKc=";
        let bbs_private_key = "WWTZW8pkz35UnvsUCEsof2CJmNHaJQ/X+B5xjWcHr/I=";
        let schema_did = "did:evan:EiACv4q04NPkNRXQzQHOEMa3r1p_uINgX75VYP2gaK5ADw";
```

Reply:
Removed r#...#